README.md

```diff
@@ -8,6 +8,30 @@ To get started with the Hypersync SDK hop on over to the [SDK documentation](doc
 
 ## Release Notes
 
+### 3.0.0
+
+- Classes, functions, types, interfaces and enums common to all integration types have been
+  moved to @hyperproof/integration-sdk.
+
+- Paging of data set results can now be done delcaratively. For more information on
```

Comment:
nit: declaratively is spelt incorrectly (sorry, I hate to put a stupid nit like this on a giant PR)

Reply:
Fixed.